src/rust/engine/Cargo.lock

```diff
@@ -501,6 +501,8 @@ source = "registry+https://github.com/rust-lang/crates.io-index"
 checksum = "c3d4260bcc2e8fc9df1eac4919a720effeb63a3f0952f5bf4944adfa18897f09"
 dependencies = [
  "memchr",
+ "once_cell",
+ "regex-automata 0.1.10",
```

Comment:
It looks like this depends on a feature of bstr that pulls in an extra version of regex-automata, since our bstr is currently very old. Is it possible/sensible to bump bstr too?

Reply:
Good point. Ran `cargo update -p bstr` and it upgraded `bstr` to v1.8.x which also removed the regex-automata dependency.